packages/ui-patterns/src/McpUrlBuilder/constants.tsx

```diff
@@ -51,6 +51,10 @@ export const FEATURE_GROUPS: McpFeatureGroup[] = [
   },
 ]
 
+export const FEATURE_GROUPS_NON_PLATFORM = FEATURE_GROUPS_PLATFORM.filter((group) =>
+  ['docs', 'database'].includes(group.id)
```

Comment:
Maybe could wire this up to the same list of constants as the query schema in the server implementation:

https://github.com/supabase/supabase/blob/90976b798848ccc5bcb9c9b67a302f51609882ad/apps/studio/pages/api/mcp/index.ts#L10